diff --git a/static/js/index.js b/static/js/index.js
index 7550a92..6e95cf8 100644
--- a/static/js/index.js
+++ b/static/js/index.js
@@ -5,11 +5,20 @@ exports.postAceInit = function(hook, context){
     return false;
   });

+  $('iframe[name="ace_outer"]').contents().find('iframe').contents().find("#innerdocbody").on('contextmenu', "a", function (e){
+    $(this).attr('rel','noreferrer');
+  });
+
   $('#chattext > p').on('click', "a", function (e){
     window.open('../redirect#'+escape(e.currentTarget.href));
     e.preventDefault();
     return false;
   });
+
+  $('#chattext > p').on('contextmenu', "a", function (e){
+    $(this).attr('rel','noreferrer');
+  });
+
 }

 exports.postTimesliderInit = function(hook, context){
@@ -18,4 +27,9 @@ exports.postTimesliderInit = function(hook, context){
     e.preventDefault();
     return false;
   });
+
+  $('#padcontent').on('contextmenu', "a", function (e){
+    $(this).attr('rel','noreferrer');
+  });
+
 }
